packages/design-system-react-native/src/components/BadgeCount/BadgeCount.constants.ts

```diff
@@ -0,0 +1,23 @@
+import { TextVariant } from '../Text';
+import { BadgeCountSize } from './BadgeCount.types';
+
+// Mappings
+export const MAP_BADGECOUNT_SIZE_TEXTVARIANT: Record<
+  BadgeCountSize,
+  TextVariant
+> = {
+  [BadgeCountSize.Md]: TextVariant.BodyXs,
+  [BadgeCountSize.Lg]: TextVariant.BodySm,
+};
+export const MAP_BADGECOUNT_SIZE_LINEHEIGHT: Record<BadgeCountSize, string> = {
+  [BadgeCountSize.Md]: 'leading-3.5', // line-height 14px
+  [BadgeCountSize.Lg]: 'leading-4', // line-height 16px
+};
+
+export const TWCLASSMAP_BADGECOUNT_SIZE_CONTAINER: Record<
+  BadgeCountSize,
+  string
+> = {
+  [BadgeCountSize.Md]: 'min-w-4 h-3.5 py-0 px-1', // min-width 16px, height 14px, padding-vertical 0, padding-horizontal 4
```

Comment:
I don't think these decimal classnames exist in tailwind spacing `h-3.5`? I think this is the cause of it breaking

<img width="72" alt="Screenshot 2025-03-21 at 1 58 51 PM" src="https://github.com/user-attachments/assets/89ecd17c-1183-4561-9ef3-0f72a0fdd954" />